### ISSUE TITLE
[Short.io] ...

### ISSUE THREAD

**Issue by daksh-gg** [raycast/extensions#15565]
### Extension

https://www.raycast.com/koinzhang/short-io

### Raycast Version

1.86.1

### macOS Version

15.1

#### Error:
```
TypeError: Cannot read properties of undefined (reading 'substring')

zs:search-links.tsx:13:49

---
10: import EditLink from "./edit-link"

### Steps To Reproduce

Use "Search links" to search through your links



### Current Behaviour

No links are found / error 
![Arc ss - QkeJnP60@2x](https://github.com/user-attachments/assets/4ab6021d-362a-47bf-82c4-a89ce2f45bf9)

### Expected Behaviour

Shows you your links

**Comment by koinzhang** [comment]
Hi @daksh-gg, thanks for your feedback, fixed in #15628

**Comment by koinzhang** [comment]
@raycastbot close this issue
